week-1/1-exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function first
-
-var result = divide(3, 4);
+function divide() {
+  return 3 / 4;
```

Comment:
Again, this is not using function parameters.  Can you figure out how this function would look using them?